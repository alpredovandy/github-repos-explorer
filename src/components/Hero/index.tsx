const Hero = () => (
    <div className="flex justify-between items-center">
        <span className="text-md font-bold text-[#2c9cdb]">Powered by Github.com</span>
        <span className="italic text-neutral-600">@alpredovandy</span>
    </div>
);

export default Hero;
