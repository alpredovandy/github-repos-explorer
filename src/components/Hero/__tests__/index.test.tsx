import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import Hero from '@/components/Hero';

describe('Test components: Hero', () => {
    const renderHero = () => render(<Hero />);

    it('Snapshot', () => {
        const { container } = renderHero();

        expect(container).toMatchSnapshot();
    });
});
