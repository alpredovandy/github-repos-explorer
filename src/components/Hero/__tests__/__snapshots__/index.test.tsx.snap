// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test components: Hero > Snapshot 1`] = `
<div>
  <div
    class="flex justify-between items-center"
  >
    <span
      class="text-md font-bold text-[#2c9cdb]"
    >
      Powered by Github.com
    </span>
    <span
      class="italic text-neutral-600"
    >
      @alpredovandy
    </span>
  </div>
</div>
`;
